src/MagnumPlugins/WavAudioImporter/WavHeader.h

```diff
@@ -31,30 +31,59 @@
 
 #include "Magnum/Types.h"
 
+#define WAVE_FORMAT_PCM 0x0001
```

Comment:
Right. Sorry, overlooked that.

The `Utility::Endianness` class should work on [`enum class` types without problems](https://github.com/mosra/corrade/commit/ed226f7519921db4cd8ca06af6be7e41a8617e2d).